distribution/metadata/blobsum_service.go

```diff
@@ -13,8 +13,14 @@ type BlobSumService struct {
 	store Store
 }
 
+// BlobSum contains the digest and source repository information for a layer.
+type BlobSum struct {
```

Comment:
I really don't want to continue to propagate the terminology of "blob sum". It is a vestige of the original proposal and it is inaccurate to use with the current schema.